content/_data/changelogs/weekly.yml

```diff
@@ -4502,6 +4502,22 @@
     # pull: 4044 irrelevant (Use the sponsor button to link to our donation page)
     # pull: 4043 too minor (Switch translations for 2 labels in iconSize_pl.properties)
     # pull: 4031 too minor (Add an example how to launch agent with secret not on command line)
+- version: '2.180'
+  date: 2019-06-09
+  changes:
+    - type: major bug
+      pull: 4047
+      issue: 57713
```

Comment:
https://issues.jenkins-ci.org/browse/JENKINS-57759 to be linked also IMHO

Reply:
Oops, missed this. Will follow up.